relay-server/src/endpoints/keda.rs

```diff
@@ -0,0 +1,25 @@
+use crate::http::StatusCode;
+use crate::service::ServiceState;
+use crate::services::keda::KedaMessageKind;
+use ahash::{HashMap, HashMapExt};
+
+/// Returns internal metrics data for relay.
+pub async fn handle(state: ServiceState) -> (StatusCode, String) {
+    let data = match state.keda().send(KedaMessageKind::Check).await {
+        Ok(data) => data,
+        Err(_) => {
+            return (
+                StatusCode::INTERNAL_SERVER_ERROR,
+                "Failed to collect internal metrics".to_string(),
+            )
+        }
+    };
+
+    match serde_prometheus::to_string(&data, None, HashMap::new()) {
```

Comment:
This adds a bunch of dependencies, maybe we can just serialize it ourselves, the format is quite simple and all of our data is pretty much static.